packaging/make-all-top.sh

```diff
@@ -43,7 +43,7 @@ if test -f ldms/src/sampler/meminfo/meminfo.c; then
 	cd $build_subdir
 	expected_ovislib_prefix=$prefix
 	expected_sos_prefix=/badsos
-	allconfig="--prefix=$prefix --disable-rdma --enable-ssl --with-libevent=$expected_event2_prefix --disable-sos --disable-perfevent --disable-rpath --enable-zap --disable-zaptest --enable-authentication --disable-sysclassib --disable-ibnet"
+	allconfig="--prefix=$prefix --disable-rdma --enable-ssl --with-libevent=$expected_event2_prefix --disable-sos --disable-perfevent --disable-rpath --enable-zap --disable-zaptest --enable-authentication --disable-sysclassib --disable-ibnet --disable-python"
```

Comment:
I don't think disable-python belongs in this patch.

Reply:
yes. will need to back out changes to make-all-top. Esp. if python bits have moved to default python disabled as needed.

Reply:
> @morrone We also noticed the tap bits in the automake docs, but lacking a credible, complete, and apparently maintainable example, we haven't yet managed to apply it. If you happen to know of a real tap deployment that is compatible with automake 1.13.4 (toss3/rhel7), we'd love to see it.

You are asserting some strong generalities that have not been true in my experience. If you want to discuss particular problems you are having in another issue, we could do that. I suspect they can all be worked through.

Flux makes extensive use of TAP. https://github.com/flux-framework/flux-core